configs/ppyoloe/ppyoloe_plus_l_fast_8xb8-80e_coco.py

```diff
@@ -0,0 +1,22 @@
+_base_ = './ppyoloe_plus_s_syncbn_fast_8xb32-300e_coco.py'
```

Comment:
Let's put all the configurations of the non-plus version, but we will not to train.